modules/samples/artifacts/PublishHttpOAuthRequest/PublishHttpOAuthRequest.siddhi

```diff
@@ -0,0 +1,81 @@
+@App:name("PublishHttpOAuthRequest")
+@App:description("Send a HTTP events to an OAuth-protected endpoint")
+/*
+Purpose:
+	This application demonstrates how to configure WSO2 Stream Processor to send a HTTP events to an OAuth-protected
+	 endpoint.
+Prerequisites:
+     1) Replace the Sink configuration values for following options.
+            - publisher.url : publisher URL (Ex:- 'https://localhost:8005/abc')
+            - consumer.key  : consumer key for the http request (Ex:- 'abcdef')
+            - consumer.secret: consumer secret for the http request (Ex:- 'abcdef')
+            - token.url     : URL of the token end point (Ex:- 'https://localhost:8005/token')
+            - method        : method type (Eg:- POST)
+
+            optional (You can fill this if it is different from default values)
+            - header (Authorization header)  : access token for the API endpoint request (Ex:- 'abcdef') <= by default
+             it automatically get the access token using the password/client-credential/refresh grant. here the grant
+             type depends on the user input.
+            - https.truststore.file : API trust store path (Ex:- '/user/../../../client-truststore.jks') <= by default
+             it get from product pack (Ex:- ${carbon.home}/resources/security/client-truststore.jks)
+            - https.truststore.password :  API trust store password (wso2carbon) <= by default it set as wso2carbon
+
+     2) Save one of the convenient sample mention below and configure the values. If there is no syntax error,
+     the following message is shown on the console:
```

Comment:
This statement is not very clear to me.
Shall we replace above with following:

```
Save this sample. If there is no syntax error, the following message is shown on the console:
```
I found above line in [1]. Please replace it in every sample file here where it is applicable. 

[1] https://github.com/wso2/product-sp/blob/master/modules/samples/artifacts/PublishHttpInXmlFormat/PublishHttpInXmlFormat.siddhi